Rx cap time out (#16203)
nrfconnect/sdk-nrf: samples/wifi/radio_test/sample_description.rst

              FDDF45
              07CF3D

>            In clean RF environment packet detection does not take place, with following output:

Reviewer: ```suggestion
           Packet detection does not take place in a clean RF environment, producing the following output:
```

              The captured samples will vary from run to run.
            * The capture is taken after WLAN packet detection, so it will not have the first few samples in the first WLAN packet.
            * Smaller packets should be used so that multiple packets can be seen in the capture.

>             * In clean environment packet detection will fail.

Reviewer 2: 1. In **RF** clean environment
 2. Instead of saying "will fail" it is better to say "does not happen". ???

              The captured samples will vary from run to run.
            * The capture is taken after WLAN packet detection, so it will not have the first few samples in the first WLAN packet.
            * Smaller packets should be used so that multiple packets can be seen in the capture.

>             * In clean RF environment packet detection does not take place.

Reviewer: ```suggestion
            * Packet detection does not take place in a clean RF environment.
```

>             * Command will timeout if no packets are detected within set timeout period. 

Reviewer: ```suggestion
            * The command will timeout if no packets are detected within set timeout period. 
```

Is the timeout period set by the user?

Author: Yes. It is set by the user.
